pkgs/development/ocaml-modules/afl-persistent/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, buildOcaml, fetchFromGitHub, ocaml, findlib, opam }:
```

Comment:
`buildOcaml` is only useful if you package jane-street libraries for OCaml 4.02.

Reply:
oh yeah, just forgot to delete that